### ISSUE TITLE
Apartment::Tenant.current returns environmentified tenant name

### ISSUE THREAD

**Issue by kuzukuzu** [influitive/apartment#465]
I think it's curious behavior, because of reasons listed below.

- It's recommended that basic operations ( `create` , `switch!` and `drop` ) are given tenant name without environment.
- It should be encapsulated if environment is prepended/appended to tenant name or not.

**Comment by mikecmpbll** [comment]
i agree. it's actually a bug tbh, because it differs between adapters. pg adapter tracks an instance variable `@current`, and others look at `connection.current_database`.

could just be me being blind(/ill) but i also can't actually see where the pg adapter does "environmentify"..
